v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -1613,5 +1613,13 @@ Please use the fully qualified name of the contract to disambiguate it.`,
 
 Please only include one of the plugins in your Hardhat configuration.`,
     },
+    DEPLOYMENT_ERROR: {
```

Comment:
This is the generic error that is used when a deployment fails (e.g. validation, timeout etc).
Previously that was a HardhatPluginError, now it is a HardhatError where the message is passed in.